### ISSUE TITLE
Incorrectly displayed trailing text

### ISSUE THREAD

**Issue by srGritsuk** [cgoldsby/TvOSMoreButton#26]
**What did you do**

Set the text more than can be reflected

**What did you expect to happen?**

The test is displayed partially, and trailing text is displayed completely

_________________________________________________________________ 
 Text text text text text text text text text text text text text te 
 xt  text text text text  text text text text  text text text text text  
 text text text text  text text text text  text text text tex...  **More**  
_________________________________________________________________

**What happened instead?**

The test is displayed partially, and trailing text is displayed partially

_________________________________________________________________
 Text text text text text text text text text text text text text te 
 xt  text text text text  text text text text  text text text text text  
 text text text text  text text text text  text text text tex...  **Mor...**  
_________________________________________________________________

**Why did it happen?**

Text and trailing text have different attributes, so you need to use NSMutableAttributedString to calculate the height.

```
func willFit(to size: CGSize,
                 ellipsesString: String = "",
                 trailingText: String = "",
                 attributes: [NSAttributedString.Key: Any],
                 trailingTextAttributes: [NSAttributedString.Key: Any]) -> Bool {

        let attributedString = NSMutableAttributedString(string: self + ellipsesString, attributes: attributes)
        let attributedTrailingString = NSAttributedString(string: trailingText, attributes: trailingTextAttributes)
        attributedString.append(attributedTrailingString)
        
        let boundedSize = CGSize(width: size.width, height: .greatestFiniteMagnitude)
        let options: NSStringDrawingOptions = [.usesLineFragmentOrigin, .usesFontLeading]
        let boundedRect = attributedString.boundingRect(with: boundedSize,
                                                        options: options,
                                                        context: nil)
        return boundedRect.height <= size.height
    }
```

**Comment by cgoldsby** [comment]
@srGritsuk, thank you for making the `TvOSMoreButton` better by submitting this excellent bug report. 🏅 

I was able to reproduce this bug using the Example project and it does seem to be related to the trailing text attributes not being used when calculating the bounding rect.

<img width="278" alt="screen shot 2018-12-17 at 8 00 54 am" src="https://user-images.githubusercontent.com/3449816/50088961-7a696c00-01d2-11e9-88b4-f938df4ecc9e.png">

Would you like to create a PR with the fix? I would love to have more contributors. If not; no worries.

**Comment by srGritsuk** [comment]
@cgoldsby, thanks for the quick response!
Of course, I would like to become a member of the project and create a PR with fix, and make the project even better in the future.

**Comment by cgoldsby** [comment]
@srGritsuk, thank you for your contribution, PR for fixing this bug, and, of course, interest in the `TvOSMoreButton`. 🍻

I published **TvOSMoreButton (1.2.2)** to CocoaPods. 🚀
